package.json

```diff
@@ -136,7 +136,8 @@
     "test": "TZ=UTC jest",
     "bump": "lerna version",
     "cypress:open": "cypress open",
-    "cypress:run": "cypress run"
+    "cypress:run": "cypress run",
+    "depcheck": "./utils/check-dependencies-exist.js"
```

Comment:
Should we set this to prebuild too? Instead of running it explicitly in Travis?

Reply:
@lychyi Just realized we don't run `yarn build` as part of Travis PR builds (meaning `prebuild` doesn't get run). Do you think we should keep it in both places?